File: WEEK 1/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void) 
{
    int height,i=1,temp=0;
    int space,count;
    do
    {
        height = get_int("Enter Height : ");
    }
    while( (height<0) || (height>20) );    
    count=height;

  for( ; i<=height ; ++i, space=0,temp=0,count-=1)
  {
      while(space!=count)
        { printf(" "); space++; } //print spaces as per needed 
      while(temp!=i)
        { printf("#"); temp++; }  //print left side pyramid 
    
      printf("  ");  //print two spaces in between pyramid
    
      temp=0;  //set temp to 0 for print right side pyramid
    
      while(temp!=i)
        {  printf("#");  temp++; } //print right side pyramid
     printf("\n"); //new line 
  }
}

File: WEEK6/mario.py
from cs50 import get_int

height = get_int("Height: ")

while height < 1 or height > 8:
    print("Invalid Height!!")
    height = get_int("Height: ")

for i in range(height):
    print((" " * (height-i-1)) + ("#" * (i+1)) + ("  ") + ("#" * (i+1)))
